libs/application/core/src/types/ApplicationTypes.ts

```diff
@@ -9,3 +9,15 @@ export enum ApplicationTypes {
   HEALTH_INSURANCE = 'HealthInsurance',
   CHILDREN_RESIDENCE_CHANGE = 'ChildrenResidenceChange',
 }
+
+export const ApplicationTranslations = {
+  ExampleForm: 'example.application',
+  Passport: null,
+  DrivingLessons: 'dl.application',
+  DrivingLicense: 'dl.application',
```

Comment:
I assume this being the same is intentional?

Reply:
Yes, they are both using the same namespace